app/controllers/apartments_controller.rb

```diff
@@ -33,15 +34,16 @@ def update
 
     private
     def apartment_params
-        params.expect(apartment: [ :unit_name,
+        params.require(:apartment).permit(:unit_name,
                                     :address,
                                     :city,
                                     :state,
                                     :bed_count,
                                     :bath_count,
                                     :sqft,
                                     :rent,
-                                    :date_available ])
+                                    :date_available,
+                                    :apartment_image)
```

Comment:
We are adding the `apartment_image` to the model here (I'm assuming for our purposes that the business is clear on only needing a single image for Apartment records).

That said, we should definitely have some test coverage for this new field prior to merge.